### ISSUE TITLE
Unnecessary "Review our 2.2 release candidate!" block

### ISSUE THREAD

**Issue by likemusic** [magento/devdocs#1505]
Feedback on page:
- /guides/v2.2/config-guide/log/log-intro.html
- /guides/v2.2/config-guide/log/log-magento.html
- /guides/v2.2/config-guide/log/log-db.html

This pages contains block with "Review our 2.2 release candidate!", which seems should be deleted.

![screenshot_6](https://user-images.githubusercontent.com/603401/32149341-df1631f6-bd13-11e7-8cfb-2b740238c353.png)

**Comment by jeff-matthews** [comment]
Good catch @likemusic!

**Comment by jeff-matthews** [comment]
This is fixed.

**Comment by likemusic** [comment]
By goole (`site:devdocs.magento.com "Review our 2.2 release candidate"`) I found many other page with the same block. Maybe some of them should exists, but I think that most of them shouldn't. Seems that redirect to v2.2 or just remove this block is more preferable for them.

List of urls from google:

- http://devdocs.magento.com/guides/v2.2/release-notes/release-notes-2-2-prerelease1-CE.html
- http://devdocs.magento.com/guides/v2.2/release-notes/release-notes-2-2-RC2-EE.html
- http://devdocs.magento.com/guides/v2.2/release-notes/release-notes-2-2-RC3-EE.html
- http://devdocs.magento.com/guides/v2.2/release-notes/release-notes-2-2-RC3-CE.html
- http://devdocs.magento.com/guides/v2.2/release-notes/release-notes-2-2-prerelease1-EE.html
- http://devdocs.magento.com/guides/v2.2/release-notes/release-notes-2-2-RC2-CE.html
- http://devdocs.magento.com/guides/v2.2/release-notes/release-candidate/quick-start.html
- http://devdocs.magento.com/guides/v2.2/release-notes/release-candidate/submit-issues.html
- http://devdocs.magento.com/guides/v2.2/release-notes/release-candidate/install.html
- http://devdocs.magento.com/guides/v2.2/architecture/software_versions.html
- http://devdocs.magento.com/guides/v2.2/architecture/versioning_overview.html
- http://devdocs.magento.com/guides/v2.2/architecture/versioning%20overview.html
- http://devdocs.magento.com/guides/v2.2/architecture/versioning_spec_format.html
- http://devdocs.magento.com/guides/v2.2/architecture/change_table.html
- http://devdocs.magento.com/guides/v2.2/architecture/stack-basics.html
- http://devdocs.magento.com/guides/v2.2/architecture/change_table.html
- http://devdocs.magento.com/guides/v2.2/architecture/version_store.html
- http://devdocs.magento.com/guides/v2.2/architecture/stack-basics.html
- http://devdocs.magento.com/guides/v2.2/architecture/archi_perspectives/components/arch_libraries.html
- http://devdocs.magento.com/guides/v2.2/architecture/archi_perspectives/components/arch_translations.html
- http://devdocs.magento.com/guides/v2.2/architecture/archi_perspectives/themes_intro.html
- http://devdocs.magento.com/guides/v2.2/architecture/arch_translations.html
- http://devdocs.magento.com/guides/v2.2/cloud/trouble/trouble-broken-branch.html
- http://devdocs.magento.com/guides/v2.2/architecture/view/static-process.html
- http://devdocs.magento.com/guides/v2.2/architecture/view/template-engine.html
- http://devdocs.magento.com/guides/v2.2/architecture/view/page-assets.html
- http://devdocs.magento.com/guides/v2.2/architecture/view/xml-schema-layout.html
- http://devdocs.magento.com/guides/v2.2/architecture/behavior/price-render.html
- http://devdocs.magento.com/guides/v2.2/config-guide/cache/caching.html

**Comment by likemusic** [comment]
I don't understand how this block generated, because I can't find string "Review our 2.2 release candidate" in repository. Find by block class "message-banner" found just in css-files.

Can anyone explain it for me?

**Comment by likemusic** [comment]
Another issue with this block is that pages with this block for users who comes from search engines are misleads. For users seems that version v2.2 still in the RC stage because it latest item in version selector at head of page.
![screenshot_7](https://user-images.githubusercontent.com/603401/32198510-6e35ae96-bdd9-11e7-82e8-f96d2472c1fb.png)

**Comment by likemusic** [comment]
When I build devdocs by vagrant on windows for page with "Review our 2.2 release candidate!" block I got "404 Not found". But why not the same on the http://devdocs.magento.com/ ?

**Comment by jeff-matthews** [comment]
Hi @likemusic, we deleted the source files for all the examples you've listed, but didn't delete the corresponding HTML output from the `gh-pages` branch as well. That's why you're not seeing them when you build locally, which is the correct and expected behavior.

We created an internal issue to refresh our `gh-pages` branch and delete all HTML files that don't have a corresponding markdown source file.
